graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/ast/expressions/AdditionExpression.java

```diff
@@ -92,6 +91,13 @@ public Object evaluateUnsafe(EvaluationContext context) {
             } else {
                 return new Duration(right, left);
             }
+        }else if(String.class.equals(type) ){
```

Comment:
Please format whitespace here according to the rest of the file (space before and after keywords, between `)` and `{` and no inner space in `()` expressions.